engine-tests/Cargo.toml

```diff
@@ -37,11 +37,11 @@ serde_json = "1"
 hex = { version = "0.4.3", default-features = false }
 near-sdk = { git = "https://github.com/aurora-is-near/near-sdk-rs.git", rev = "5e58722bd61d9d24ae6293326146c751f0a814fb" }
 near-sdk-sim = { git = "https://github.com/aurora-is-near/near-sdk-rs.git", rev = "5e58722bd61d9d24ae6293326146c751f0a814fb" }
```

Comment:
Not that this intentionally doesn't update the sdk. It already is the case that nearcore used by sdk is different from the nearcore used by engine-tests (that is, there are already two versions of `near-vm-runner` in Cargo.lock today).